src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -16,7 +16,7 @@ const getPropertyValue = require('../../../../../_common/utility').getPropertyVa
 const PersonalDetails = (() => {
     const form_id                = '#frmPersonalDetails';
```

Comment:
```suggestion
    const form_id              = '#frmPersonalDetails';
```

Reply:
Can we align all equal signs on top of each other like it was before? In binary, we do this for better code readability across all variable assignments. The number of spaces is based on the longest variable name in the group, the one you changed previously in this case.

Reply:
@akmal-binary That's weird, I didn't touch this line at all.. Prettifier did it perhaps. Will change it back now.